docs/sdks/iOS.mdx

```diff
@@ -435,5 +438,31 @@ cancellable = VitalHealthKitClient.shared.status.sink { value in
 }
 ```
 
+### 1. Background Delivery
+
+You can enable the SDK to deliver HealthKit data in background. This provides a more seamlessly experience, since the user doesn't need to open your app to sync data. Start by enabling "Background Delivery" in the app capabilities:
```

Comment:
```suggestion
You can enable the SDK to deliver HealthKit data in background. This provides a more seamless experience, since the user doesn't need to open your app to sync data. Start by enabling "Background Delivery" in the app capabilities:
```